benchmarks/scripts/cache_cleaning_workers.py

```diff
@@ -0,0 +1,114 @@
+from mysql_utils import create_connection
```

Comment:
Where is this being called?

Reply:
In https://github.com/prestodb/pbench/pull/32/files#diff-178b21505ceae58580985f1d1c6c27f05c66411c197c31a5efd7ca0acc94dd32R74 to get the worker list from mysql